problems/[230731] 55. Jump Game.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * @param {number[]} nums
+ * @return {boolean}
+ */
+var canJump = function (nums) {
+  let index = 0;
+  let afterMaxJump = 0;
+  const lastIndex = nums.length - 1;
+
+  if (!lastIndex) return true;
```

Comment:
```suggestion
  const lastIndex = nums.length - 1;

  if (!lastIndex) return true;
  
  let index = 0;
  let afterMaxJump = 0;
```

조건문을 `index` `afterMaxJump` 변수 선언 이전에 위치시키면 불필요한 연산을 조금 더 방지할 수 있을것 같습니닷!